src/coach/scripts/coach.py

```diff
@@ -1,23 +1,36 @@
 #!/usr/bin/env python
 
 import rospy
+from robocup_msgs.msg import *
 
 def coach():
```

Comment:
Pascal case

Reply:
So RoboCup_Msgs instead?

Reply:
oh, I though coach was a class 😃 . It's ok